web_responsive/static/src/legacy/scss/web_responsive.scss

```diff
@@ -459,3 +459,8 @@ html .o_web_client .o_action_manager .o_action {
         padding-right: 0;
     }
 }
+
+// Many2one li items with wrap
+.dropdown-item {
+    white-space: initial;
```

Comment:
Too broad. Apply it just to the children of the relational fields dropdown dropdown

Reply:
Changed!